run.js

```diff
@@ -93,6 +93,9 @@ function startTraefik() {
   }
   if (process.env.HTTPS) {
     flags.push("--entrypoints.websecure.address=:443")
+    // Redirect http -> https
```

Comment:
I didn't test, but found this documentation link that's convincing: https://doc.traefik.io/traefik/routing/entrypoints/#redirection.

Reply:
added that link to comment